2020-07-16 17:12:56,884 ERROR frappe New Exception collected with id: 2020-07-16 17:12:56.399511-127.0.0.1-d63
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-16 17:12:56,884 ERROR frappe New Exception collected with id: 2020-07-16 17:12:56.401846-127.0.0.1-b75
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-17 14:07:31,614 ERROR frappe New Exception collected with id: 2020-07-17 14:07:31.457428-127.0.0.1-485
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Module Def","name":"New Module Def 1","__islocal":1,"__unsaved":1,"owner":"Administrator","module_name":"Extracurricular Activities","app_name":"Extracurricular Activities"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-17 14:07:45,361 ERROR frappe New Exception collected with id: 2020-07-17 14:07:45.338001-127.0.0.1-d4c
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Module Def","name":"New Module Def 1","__islocal":1,"__unsaved":1,"owner":"Administrator","module_name":"Extracurricular Activities","app_name":"Extracurricular Activities"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-17 14:09:28,884 ERROR frappe New Exception collected with id: 2020-07-17 14:09:28.850826-127.0.0.1-ccf
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Module Def","name":"New Module Def 2","__islocal":1,"__unsaved":1,"owner":"Administrator","__run_link_triggers":1,"module_name":"aplliiii","app_name":"ddda"}', 'cmd': 'frappe.client.save'}
2020-07-20 11:29:37,874 ERROR frappe New Exception collected with id: 2020-07-20 11:29:37.724073-127.0.0.1-222
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-20 14:18:08,327 ERROR frappe New Exception collected with id: 2020-07-20 14:18:07.600407-127.0.0.1-27a
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-20 14:18:08,328 ERROR frappe New Exception collected with id: 2020-07-20 14:18:07.599352-127.0.0.1-f2d
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-20 14:30:53,973 ERROR frappe New Exception collected with id: 2020-07-20 14:30:53.310702-127.0.0.1-b03
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-20 14:30:53,975 ERROR frappe New Exception collected with id: 2020-07-20 14:30:53.308999-127.0.0.1-6e8
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-20 14:30:53,975 ERROR frappe New Exception collected with id: 2020-07-20 14:30:53.310015-127.0.0.1-846
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:22:59,470 ERROR frappe New Exception collected with id: 2020-07-21 16:22:59.057365-127.0.0.1-13d
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:22:59,470 ERROR frappe New Exception collected with id: 2020-07-21 16:22:59.057790-127.0.0.1-929
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:22:59,470 ERROR frappe New Exception collected with id: 2020-07-21 16:22:59.057054-127.0.0.1-f69
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:30:45,709 ERROR frappe New Exception collected with id: 2020-07-21 16:30:45.462629-127.0.0.1-110
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:30:45,713 ERROR frappe New Exception collected with id: 2020-07-21 16:30:45.482467-127.0.0.1-afc
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:40:07,148 ERROR frappe New Exception collected with id: 2020-07-21 16:40:07.005413-127.0.0.1-ada
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection Action', 'name': 'a9db42345b', 'cmd': 'frappe.desk.form.load.getdoc'}
2020-07-21 16:43:31,073 ERROR frappe New Exception collected with id: 2020-07-21 16:43:30.631941-127.0.0.1-e44
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 16:43:31,073 ERROR frappe New Exception collected with id: 2020-07-21 16:43:30.632213-127.0.0.1-c57
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 17:46:19,395 ERROR frappe New Exception collected with id: 2020-07-21 17:46:18.958423-127.0.0.1-197
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-21 17:46:19,401 ERROR frappe New Exception collected with id: 2020-07-21 17:46:18.958768-127.0.0.1-ebf
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 09:50:24,836 ERROR frappe New Exception collected with id: 2020-07-22 09:50:24.410553-127.0.0.1-e22
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 09:50:24,842 ERROR frappe New Exception collected with id: 2020-07-22 09:50:24.411160-127.0.0.1-b29
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 09:50:24,842 ERROR frappe New Exception collected with id: 2020-07-22 09:50:24.410914-127.0.0.1-1ea
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 11:06:33,678 ERROR frappe New Exception collected with id: 2020-07-22 11:06:33.516739-127.0.0.1-59b
Site: erpnext_dev
Form Dict: {'docs': '{"docstatus":0,"doctype":"Quality Rejection Action","name":"New Quality Rejection Action 1","__islocal":1,"__unsaved":1,"owner":"Administrator","correctivepreventive":"Corrective","status":"Open"}', 'method': 'Quality Action', 'cmd': 'runserverobj'}
2020-07-22 12:40:58,155 ERROR frappe New Exception collected with id: 2020-07-22 12:40:57.741050-127.0.0.1-012
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 12:40:58,157 ERROR frappe New Exception collected with id: 2020-07-22 12:40:57.741593-127.0.0.1-49c
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 12:40:58,157 ERROR frappe New Exception collected with id: 2020-07-22 12:40:57.742031-127.0.0.1-51c
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 12:54:12,469 ERROR frappe New Exception collected with id: 2020-07-22 12:54:12.053388-127.0.0.1-20f
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 15:42:26,013 ERROR frappe New Exception collected with id: 2020-07-22 15:42:25.613940-127.0.0.1-2aa
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 15:42:26,018 ERROR frappe New Exception collected with id: 2020-07-22 15:42:25.612254-127.0.0.1-749
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-22 15:42:26,019 ERROR frappe New Exception collected with id: 2020-07-22 15:42:25.611737-127.0.0.1-d1f
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-23 14:56:31,960 ERROR frappe New Exception collected with id: 2020-07-23 14:56:31.823686-127.0.0.1-dd9
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-23 15:14:42,284 ERROR frappe New Exception collected with id: 2020-07-23 15:14:41.807763-127.0.0.1-65f
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-23 15:14:42,289 ERROR frappe New Exception collected with id: 2020-07-23 15:14:41.811673-127.0.0.1-03a
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-24 12:00:55,916 ERROR frappe New Exception collected with id: 2020-07-24 12:00:55.755390-127.0.0.1-330
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'workflow_state_field': 'workflow_state', 'states': '["Draft","Approved By Qality Manager"]', 'cmd': 'frappe.workflow.doctype.workflow.workflow.get_workflow_state_count'}
2020-07-27 15:20:39,835 ERROR frappe New Exception collected with id: 2020-07-27 15:20:39.453276-127.0.0.1-660
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-27 15:20:39,836 ERROR frappe New Exception collected with id: 2020-07-27 15:20:39.454018-127.0.0.1-bed
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-27 15:20:39,836 ERROR frappe New Exception collected with id: 2020-07-27 15:20:39.453562-127.0.0.1-f92
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-27 18:20:34,934 ERROR frappe New Exception collected with id: 2020-07-27 18:20:34.775377-127.0.0.1-30d
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-29 10:40:15,107 ERROR frappe New Exception collected with id: 2020-07-29 10:40:14.968533-127.0.0.1-bef
Site: erpnext_dev
Form Dict: {'docs': '{"docstatus":0,"doctype":"Stock Entry","name":"New Stock Entry 1","__islocal":1,"__unsaved":1,"owner":"Administrator","naming_series":"MAT-STE-.YYYY.-","purpose":"Manufacture","company":"insphere","posting_date":"2020-07-29","posting_time":"10:40:05","set_posting_time":0,"inspection_required":0,"from_bom":0,"use_multi_level_bom":1,"is_opening":"No","items":[{"docstatus":0,"doctype":"Stock Entry Detail","name":"New Stock Entry Detail 1","__islocal":1,"__unsaved":1,"owner":"Administrator","set_basic_rate_manually":0,"stock_uom":"Nos","retain_sample":0,"cost_center":"Main - I","allow_zero_valuation_rate":0,"allow_alternative_item":0,"parent":"New Stock Entry 1","parentfield":"items","parenttype":"Stock Entry","idx":1,"expense_account":"Stock Adjustment - I"}],"expense_account":"Stock Adjustment - I","address_display":null,"purchase_receipt_no":null,"supplier_address":null,"supplier_name":null,"supplier":null,"sales_invoice_no":null,"delivery_note_no":null,"customer_address":null,"customer_name":null,"customer":null,"stock_entry_type":"Manufacture","work_order":"MFG-WO-2020-00001"}', 'method': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty', 'cmd': 'runserverobj'}
2020-07-29 10:42:18,932 ERROR frappe New Exception collected with id: 2020-07-29 10:42:18.802664-127.0.0.1-825
Site: erpnext_dev
Form Dict: {'docs': '{"docstatus":0,"doctype":"Stock Entry","name":"New Stock Entry 3","__islocal":1,"__unsaved":1,"owner":"Administrator","naming_series":"MAT-STE-.YYYY.-","purpose":"Manufacture","company":"insphere","posting_date":"2020-07-29","posting_time":"10:42:11","set_posting_time":0,"inspection_required":0,"from_bom":0,"use_multi_level_bom":1,"is_opening":"No","items":[{"docstatus":0,"doctype":"Stock Entry Detail","name":"New Stock Entry Detail 3","__islocal":1,"__unsaved":1,"owner":"Administrator","set_basic_rate_manually":0,"stock_uom":"Nos","retain_sample":0,"cost_center":"Main - I","allow_zero_valuation_rate":0,"allow_alternative_item":0,"parent":"New Stock Entry 3","parentfield":"items","parenttype":"Stock Entry","idx":1,"expense_account":"Stock Adjustment - I"}],"expense_account":"Stock Adjustment - I","address_display":null,"purchase_receipt_no":null,"supplier_address":null,"supplier_name":null,"supplier":null,"sales_invoice_no":null,"delivery_note_no":null,"customer_address":null,"customer_name":null,"customer":null,"stock_entry_type":"Manufacture","work_order":"MFG-WO-2020-00001"}', 'method': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty', 'cmd': 'runserverobj'}
2020-07-29 10:47:26,098 ERROR frappe New Exception collected with id: 2020-07-29 10:47:25.962606-127.0.0.1-38d
Site: erpnext_dev
Form Dict: {'docs': '{"docstatus":0,"doctype":"Stock Entry","name":"New Stock Entry 4","__islocal":1,"__unsaved":1,"owner":"Administrator","naming_series":"MAT-STE-.YYYY.-","purpose":"Manufacture","company":"insphere","posting_date":"2020-07-29","posting_time":"10:47:19","set_posting_time":0,"inspection_required":0,"from_bom":0,"use_multi_level_bom":1,"is_opening":"No","items":[{"docstatus":0,"doctype":"Stock Entry Detail","name":"New Stock Entry Detail 4","__islocal":1,"__unsaved":1,"owner":"Administrator","set_basic_rate_manually":0,"stock_uom":"Nos","retain_sample":0,"cost_center":"Main - I","allow_zero_valuation_rate":0,"allow_alternative_item":0,"parent":"New Stock Entry 4","parentfield":"items","parenttype":"Stock Entry","idx":1,"expense_account":"Stock Adjustment - I"}],"expense_account":"Stock Adjustment - I","address_display":null,"purchase_receipt_no":null,"supplier_address":null,"supplier_name":null,"supplier":null,"sales_invoice_no":null,"delivery_note_no":null,"customer_address":null,"customer_name":null,"customer":null,"stock_entry_type":"Manufacture","work_order":"MFG-WO-2020-00001"}', 'method': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty', 'cmd': 'runserverobj'}
2020-07-29 10:55:43,234 ERROR frappe New Exception collected with id: 2020-07-29 10:55:43.168246-127.0.0.1-5d3
Site: erpnext_dev
Form Dict: {'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 10:57:06,617 ERROR frappe New Exception collected with id: 2020-07-29 10:57:06.487721-127.0.0.1-681
Site: erpnext_dev
Form Dict: {'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 11:24:37,069 ERROR frappe New Exception collected with id: 2020-07-29 11:24:36.929633-127.0.0.1-fda
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 11:57:13,630 ERROR frappe New Exception collected with id: 2020-07-29 11:57:13.449519-127.0.0.1-4b5
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 11:57:35,509 ERROR frappe New Exception collected with id: 2020-07-29 11:57:35.505923-127.0.0.1-46a
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 12:06:53,015 ERROR frappe New Exception collected with id: 2020-07-29 12:06:52.872258-127.0.0.1-e19
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 12:08:16,057 ERROR frappe New Exception collected with id: 2020-07-29 12:08:15.872638-127.0.0.1-622
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 12:09:01,085 ERROR frappe New Exception collected with id: 2020-07-29 12:09:00.921701-127.0.0.1-14b
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 12:16:02,395 ERROR frappe New Exception collected with id: 2020-07-29 12:16:02.162501-127.0.0.1-600
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 12:16:40,471 ERROR frappe New Exception collected with id: 2020-07-29 12:16:40.326476-127.0.0.1-0cb
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 12:45:17,457 ERROR frappe New Exception collected with id: 2020-07-29 12:45:17.255362-127.0.0.1-c97
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 14:09:32,759 ERROR frappe New Exception collected with id: 2020-07-29 14:09:32.622388-127.0.0.1-734
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 14:10:10,308 ERROR frappe New Exception collected with id: 2020-07-29 14:10:10.102287-127.0.0.1-2d3
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-29 14:20:27,610 ERROR frappe New Exception collected with id: 2020-07-29 14:20:27.129546-127.0.0.1-c39
Site: erpnext_dev
Form Dict: {'work_order': 'MFG-WO-2020-00001', 'cmd': 'erpnext.quality_management.doctype.quality_rejection.quality_rejection.regected_quantyty'}
2020-07-30 09:58:34,567 ERROR frappe New Exception collected with id: 2020-07-30 09:58:34.154978-127.0.0.1-7b2
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-30 10:49:06,449 ERROR frappe New Exception collected with id: 2020-07-30 10:49:06.313489-127.0.0.1-66a
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00002","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":40,"work_in_progress_warehouse":"Stores - I","rejection_quantity":20}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-30 10:50:48,092 ERROR frappe New Exception collected with id: 2020-07-30 10:50:47.912987-127.0.0.1-646
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-30 10:53:28,831 ERROR frappe New Exception collected with id: 2020-07-30 10:53:28.650000-127.0.0.1-ca5
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-30 10:54:13,300 ERROR frappe New Exception collected with id: 2020-07-30 10:54:13.143324-127.0.0.1-14e
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-30 11:01:27,563 ERROR frappe New Exception collected with id: 2020-07-30 11:01:27.400367-127.0.0.1-900
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-30 11:02:43,662 ERROR frappe New Exception collected with id: 2020-07-30 11:02:43.528265-127.0.0.1-ed3
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-30 17:06:18,199 ERROR frappe New Exception collected with id: 2020-07-30 17:06:17.758031-127.0.0.1-290
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 16:32:04.836568', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-30 17:06:30,366 ERROR frappe New Exception collected with id: 2020-07-30 17:06:30.250522-127.0.0.1-662
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 16:32:04.836568', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-30 17:06:45,302 ERROR frappe New Exception collected with id: 2020-07-30 17:06:45.262337-127.0.0.1-f9a
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 07:22:52,409 ERROR frappe New Exception collected with id: 2020-07-31 07:22:51.934285-127.0.0.1-069
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-31 07:22:52,414 ERROR frappe New Exception collected with id: 2020-07-31 07:22:51.933915-127.0.0.1-49c
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-07-31 10:58:27,313 ERROR frappe New Exception collected with id: 2020-07-31 10:58:27.164667-127.0.0.1-aa6
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":13}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:11:02,260 ERROR frappe New Exception collected with id: 2020-07-31 15:11:02.120733-127.0.0.1-945
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":14}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:14:34,783 ERROR frappe New Exception collected with id: 2020-07-31 15:14:34.625802-127.0.0.1-348
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":13}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:17:08,365 ERROR frappe New Exception collected with id: 2020-07-31 15:17:08.211639-127.0.0.1-285
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:24:40,529 ERROR frappe New Exception collected with id: 2020-07-31 15:24:40.522246-127.0.0.1-a07
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":14}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:31:13,105 ERROR frappe New Exception collected with id: 2020-07-31 15:31:12.936360-127.0.0.1-1aa
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":13}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:35:40,373 ERROR frappe New Exception collected with id: 2020-07-31 15:35:40.223679-127.0.0.1-ceb
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":13}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:36:55,122 ERROR frappe New Exception collected with id: 2020-07-31 15:36:54.974365-127.0.0.1-bd6
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":44}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:38:41,644 ERROR frappe New Exception collected with id: 2020-07-31 15:38:41.037816-127.0.0.1-acf
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00002","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":40,"work_in_progress_warehouse":"Stores - I","rejection_quantity":13,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:45:09,664 ERROR frappe New Exception collected with id: 2020-07-31 15:45:09.655227-127.0.0.1-366
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","account":"Rejection Account - I","rejection_quantity":44}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:46:01,430 ERROR frappe New Exception collected with id: 2020-07-31 15:46:01.261577-127.0.0.1-0de
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:54:59,401 ERROR frappe New Exception collected with id: 2020-07-31 15:54:59.239695-127.0.0.1-d34
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00002","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":40,"work_in_progress_warehouse":"Stores - I","rejection_quantity":6,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 15:57:29,455 ERROR frappe New Exception collected with id: 2020-07-31 15:57:29.287806-127.0.0.1-921
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00002","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":40,"work_in_progress_warehouse":"Stores - I","rejection_quantity":6,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 16:02:29,883 ERROR frappe New Exception collected with id: 2020-07-31 16:02:29.714237-127.0.0.1-35c
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00004","item_to_manufacture":"00002","item_name":"book","bom_no":"BOM-00002-001","qty_to_manufacture":100,"work_in_progress_warehouse":"Stores - I","rejection_quantity":7,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 16:04:28,308 ERROR frappe New Exception collected with id: 2020-07-31 16:04:27.983241-127.0.0.1-f0d
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 17:13:57.256488', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 16:14:36,913 ERROR frappe New Exception collected with id: 2020-07-31 16:14:36.707270-127.0.0.1-48f
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 17:13:57.256488', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 16:14:50,266 ERROR frappe New Exception collected with id: 2020-07-31 16:14:50.095819-127.0.0.1-062
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 17:13:57.256488', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 16:16:35,684 ERROR frappe New Exception collected with id: 2020-07-31 16:16:35.519104-127.0.0.1-777
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 17:13:57.256488', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 16:17:37,741 ERROR frappe New Exception collected with id: 2020-07-31 16:17:37.550380-127.0.0.1-8bf
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00002","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":40,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 16:18:54,953 ERROR frappe New Exception collected with id: 2020-07-31 16:18:54.799998-127.0.0.1-9eb
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00002","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":40,"work_in_progress_warehouse":"Stores - I","rejection_quantity":2,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 16:28:23,873 ERROR frappe New Exception collected with id: 2020-07-31 16:28:23.690590-127.0.0.1-c63
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"IT001","item_name":"Mobile","bom_no":"BOM-IT001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 17:21:35,928 ERROR frappe New Exception collected with id: 2020-07-31 17:21:35.757097-127.0.0.1-cc9
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00003","item_to_manufacture":"MAC001","item_name":"PC","bom_no":"BOM-MAC001-001","qty_to_manufacture":120,"work_in_progress_warehouse":"Stores - I","rejection_quantity":44,"account":"Rejection Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 17:56:30,549 ERROR frappe New Exception collected with id: 2020-07-31 17:56:30.374817-127.0.0.1-7ab
Site: erpnext_dev
Form Dict: {'doctype': 'Stock Entry', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:07:09,171 ERROR frappe New Exception collected with id: 2020-07-31 18:07:09.160734-127.0.0.1-5ad
Site: erpnext_dev
Form Dict: {'doctype': 'Work Order', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:07:31,261 ERROR frappe New Exception collected with id: 2020-07-31 18:07:31.255087-127.0.0.1-bd1
Site: erpnext_dev
Form Dict: {'doctype': 'Work Order', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:20:02,261 ERROR frappe New Exception collected with id: 2020-07-31 18:20:01.909832-127.0.0.1-e43
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 17:13:57.256488', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:20:11,443 ERROR frappe New Exception collected with id: 2020-07-31 18:20:11.151275-127.0.0.1-ad8
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-07-30 17:13:57.256488', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:20:17,142 ERROR frappe New Exception collected with id: 2020-07-31 18:20:17.001247-127.0.0.1-6c4
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:20:58,318 ERROR frappe New Exception collected with id: 2020-07-31 18:20:57.932997-127.0.0.1-2f6
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:24:14,250 ERROR frappe New Exception collected with id: 2020-07-31 18:24:14.090462-127.0.0.1-3a8
Site: erpnext_dev
Form Dict: {'doctype': 'Work Order', 'name': 'MFG-WO-2020-00001', 'cmd': 'frappe.client.get'}
2020-07-31 18:25:05,460 ERROR frappe New Exception collected with id: 2020-07-31 18:25:05.453103-127.0.0.1-159
Site: erpnext_dev
Form Dict: {'doctype': 'Work Order', 'name': 'MFG-WO-2020-00002', 'cmd': 'frappe.client.get'}
2020-07-31 18:28:32,768 ERROR frappe New Exception collected with id: 2020-07-31 18:28:32.760962-127.0.0.1-3c8
Site: erpnext_dev
Form Dict: {'doctype': 'Work Order', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:28:39,755 ERROR frappe New Exception collected with id: 2020-07-31 18:28:39.701675-127.0.0.1-015
Site: erpnext_dev
Form Dict: {'doctype': 'Work Order', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-07-31 18:28:53,409 ERROR frappe New Exception collected with id: 2020-07-31 18:28:53.396639-127.0.0.1-c9a
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 18:29:42,406 ERROR frappe New Exception collected with id: 2020-07-31 18:29:42.259834-127.0.0.1-a32
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-07-31 18:29:58,409 ERROR frappe New Exception collected with id: 2020-07-31 18:29:58.251199-127.0.0.1-fad
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","rejection_quantity":1}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 10:12:26,760 ERROR frappe New Exception collected with id: 2020-08-01 10:12:26.579931-127.0.0.1-477
Site: erpnext_dev
Form Dict: {'doctype': 'Stock Entry', 'with_parent': '1', 'cached_timestamp': '', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-01 10:15:32,056 ERROR frappe New Exception collected with id: 2020-08-01 10:15:31.905441-127.0.0.1-ef2
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Item Category","name":"New Item Category 1","__islocal":1,"__unsaved":1,"owner":"Administrator","category_name":"all"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 10:43:37,022 ERROR frappe Could not take error snapshot: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
Site: erpnext_dev
Form Dict: {'doctype': 'Purchase Receipt', 'filters': '{"company":["like","%insphere%"],"status":"To Bill"}', 'fields': '["count( `tabPurchase Receipt`.`name`) AS total_count"]'}
Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 136, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 51, in __init__
    frappe.local.login_manager = LoginManager()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 113, in __init__
    self.make_session(resume=True)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 194, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 197, in __init__
    self.resume()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 264, in resume
    validate_ip_address(self.user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 413, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 728, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 771, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 71, in get_doc
    return controller(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 135, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 92, in get_snapshot
    vars = cgitb.scanvars(reader, frame, locals)
  File "/usr/lib/python3.6/cgitb.py", line 88, in scanvars
    value = getattr(parent, token, __UNDEF__)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
2020-08-01 10:43:37,023 ERROR frappe Could not take error snapshot: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
Site: erpnext_dev
Form Dict: {'doctype': 'Item', 'filters': '{"disabled":0}', 'fields': '["count( `tabItem`.`name`) AS total_count"]'}
Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 136, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 51, in __init__
    frappe.local.login_manager = LoginManager()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 113, in __init__
    self.make_session(resume=True)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 194, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 197, in __init__
    self.resume()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 264, in resume
    validate_ip_address(self.user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 413, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 728, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 771, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 71, in get_doc
    return controller(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 135, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 92, in get_snapshot
    vars = cgitb.scanvars(reader, frame, locals)
  File "/usr/lib/python3.6/cgitb.py", line 88, in scanvars
    value = getattr(parent, token, __UNDEF__)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
2020-08-01 10:43:37,037 ERROR frappe Could not take error snapshot: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
Site: erpnext_dev
Form Dict: {'doctype': 'Material Request', 'filters': '{"company":["like","%insphere%"],"status":"Pending"}', 'fields': '["count( `tabMaterial Request`.`name`) AS total_count"]'}
Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 136, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 51, in __init__
    frappe.local.login_manager = LoginManager()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 113, in __init__
    self.make_session(resume=True)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 194, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 197, in __init__
    self.resume()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 264, in resume
    validate_ip_address(self.user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 413, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 728, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 771, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 71, in get_doc
    return controller(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 135, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 92, in get_snapshot
    vars = cgitb.scanvars(reader, frame, locals)
  File "/usr/lib/python3.6/cgitb.py", line 88, in scanvars
    value = getattr(parent, token, __UNDEF__)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
2020-08-01 10:43:37,038 ERROR frappe Could not take error snapshot: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
Site: erpnext_dev
Form Dict: {'doctype': 'Delivery Note', 'filters': '{"company":["like","%insphere%"],"status":"To Bill"}', 'fields': '["count( `tabDelivery Note`.`name`) AS total_count"]'}
Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 136, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 51, in __init__
    frappe.local.login_manager = LoginManager()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 113, in __init__
    self.make_session(resume=True)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 194, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 197, in __init__
    self.resume()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 264, in resume
    validate_ip_address(self.user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 413, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 728, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 771, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 71, in get_doc
    return controller(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 135, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 92, in get_snapshot
    vars = cgitb.scanvars(reader, frame, locals)
  File "/usr/lib/python3.6/cgitb.py", line 88, in scanvars
    value = getattr(parent, token, __UNDEF__)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
2020-08-01 10:43:47,503 ERROR frappe Could not take error snapshot: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
Site: erpnext_dev
Form Dict: {'doctype': 'Route History', 'records': '[{"user":"Administrator","creation":"2020-08-01 10:43:36","route":"workspace/Stock"}]'}
Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/app.py", line 136, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 51, in __init__
    frappe.local.login_manager = LoginManager()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 113, in __init__
    self.make_session(resume=True)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 194, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 197, in __init__
    self.resume()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/sessions.py", line 264, in resume
    validate_ip_address(self.user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/auth.py", line 413, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 728, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 771, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 71, in get_doc
    return controller(*args, **kwargs)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 135, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/utils/error.py", line 92, in get_snapshot
    vars = cgitb.scanvars(reader, frame, locals)
  File "/usr/lib/python3.6/cgitb.py", line 88, in scanvars
    value = getattr(parent, token, __UNDEF__)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/base_document.py", line 78, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/__init__.py", line 796, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 83, in __init__
    super(Meta, self).__init__("DocType", doctype)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 106, in __init__
    self.load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/meta.py", line 88, in load_from_db
    super(Meta, self).load_from_db()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/model/document.py", line 147, in load_from_db
    d = frappe.db.get_value(self.doctype, self.name, "*", as_dict=1)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 408, in get_value
    order_by, cache=cache)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 452, in get_values
    out = self._get_values_from_table(fields, filters, doctype, as_dict, debug, order_by, update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 599, in _get_values_from_table
    as_dict=as_dict, debug=debug, update=update)
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 122, in sql
    self.connect()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/database.py", line 75, in connect
    self._conn = self.get_connection()
  File "/home/frappe01/ERPNext/apps/frappe/frappe/database/mariadb/database.py", line 91, in get_connection
    local_infile = frappe.conf.local_infile)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/__init__.py", line 94, in Connect
    return Connection(*args, **kwargs)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 325, in __init__
    self.connect()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 599, in connect
    self._request_authentication()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 861, in _request_authentication
    auth_packet = self._read_packet()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe01/ERPNext/env/lib/python3.6/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1045, "Access denied for user '_faa3735edf87bd2f'@'localhost' (using password: YES)")
2020-08-01 11:50:28,444 ERROR frappe New Exception collected with id: 2020-08-01 11:50:28.280949-127.0.0.1-fb7
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 12:15:06,295 ERROR frappe New Exception collected with id: 2020-08-01 12:15:06.013909-127.0.0.1-821
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","comapny":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 12:17:45,029 ERROR frappe New Exception collected with id: 2020-08-01 12:17:44.873509-127.0.0.1-836
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 14:34:40,293 ERROR frappe New Exception collected with id: 2020-08-01 14:34:40.151684-127.0.0.1-3db
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 14:44:56,180 ERROR frappe New Exception collected with id: 2020-08-01 14:44:55.837850-127.0.0.1-c51
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-08-01 12:16:23.278083', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-01 15:01:36,846 ERROR frappe New Exception collected with id: 2020-08-01 15:01:36.624651-127.0.0.1-838
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-08-01 12:16:23.278083', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-01 15:12:38,676 ERROR frappe New Exception collected with id: 2020-08-01 15:12:38.279358-127.0.0.1-22d
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-08-01 12:16:23.278083', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-01 15:13:09,740 ERROR frappe New Exception collected with id: 2020-08-01 15:13:09.542324-127.0.0.1-9ab
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 15:14:12,438 ERROR frappe New Exception collected with id: 2020-08-01 15:14:12.249378-127.0.0.1-b0d
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 15:15:27,613 ERROR frappe New Exception collected with id: 2020-08-01 15:15:27.093937-127.0.0.1-367
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-08-01 12:16:23.278083', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-01 15:15:53,206 ERROR frappe New Exception collected with id: 2020-08-01 15:15:52.831316-127.0.0.1-5ec
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-08-01 12:16:23.278083', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-01 15:20:03,718 ERROR frappe New Exception collected with id: 2020-08-01 15:20:03.524941-127.0.0.1-7f5
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":13}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-01 15:20:47,104 ERROR frappe New Exception collected with id: 2020-08-01 15:20:46.960263-127.0.0.1-379
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","rejection_quantity":13}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 15:11:31,556 ERROR frappe New Exception collected with id: 2020-08-03 15:11:31.348627-127.0.0.1-a79
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":12}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 15:12:27,013 ERROR frappe New Exception collected with id: 2020-08-03 15:12:26.770851-127.0.0.1-9cf
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":2}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 16:22:20,961 ERROR frappe New Exception collected with id: 2020-08-03 16:22:20.593307-127.0.0.1-780
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'with_parent': '1', 'cached_timestamp': '2020-08-03 16:10:43.263361', 'cmd': 'frappe.desk.form.load.getdoctype'}
2020-08-03 16:43:04,356 ERROR frappe New Exception collected with id: 2020-08-03 16:43:04.200117-127.0.0.1-0f4
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 2","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","warehouse":"Scrap - I","rejection_quantity":10}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 17:12:12,783 ERROR frappe New Exception collected with id: 2020-08-03 17:12:12.625203-127.0.0.1-fd7
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","nn_account":"Stock In Hand - I","account":"Scrap Account - I","rejection_quantity":144}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 18:14:51,932 ERROR frappe New Exception collected with id: 2020-08-03 18:14:51.798922-127.0.0.1-d3b
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":11}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 18:18:48,568 ERROR frappe New Exception collected with id: 2020-08-03 18:18:48.404788-127.0.0.1-36c
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","rejection_quantity":11,"nn_account":"Scrap Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 18:21:39,568 ERROR frappe New Exception collected with id: 2020-08-03 18:21:39.374977-127.0.0.1-10b
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","nn_account":"Scrap Account - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 19:06:26,153 ERROR frappe New Exception collected with id: 2020-08-03 19:06:25.961346-127.0.0.1-9aa
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 3","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","nn_account":"Scrap Account - I","account":"Main - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 19:06:43,582 ERROR frappe New Exception collected with id: 2020-08-03 19:06:43.376552-127.0.0.1-2a4
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 3","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","nn_account":"Scrap Account - I","account":"Main - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 19:08:52,212 ERROR frappe New Exception collected with id: 2020-08-03 19:08:52.054142-127.0.0.1-78a
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 3","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","nn_account":"Scrap Account - I","account":"Main - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 19:09:12,531 ERROR frappe New Exception collected with id: 2020-08-03 19:09:12.378778-127.0.0.1-4ef
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 3","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"insphere","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"El01020001","item_name":"HP","bom_no":"BOM-El01020001-001","qty_to_manufacture":500,"work_in_progress_warehouse":"Stores - I","nn_account":"Scrap Account - I","account":"Main - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-03 21:47:36,451 ERROR frappe New Exception collected with id: 2020-08-03 21:47:36.028376-127.0.0.1-6d3
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
2020-08-04 08:25:23,488 ERROR frappe New Exception collected with id: 2020-08-04 08:25:23.322893-127.0.0.1-f73
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"in","select_work_order_number":"MFG-WO-2020-00005","item_to_manufacture":"A01020002","item_name":"ABC","bom_no":"BOM-A01020002-001","qty_to_manufacture":1000,"work_in_progress_warehouse":"Stores - I","nn_account":"Scrap Account - I","rejection_quantity":12,"account":"Stock In Hand - I"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-04 10:55:00,575 ERROR frappe New Exception collected with id: 2020-08-04 10:55:00.418054-127.0.0.1-139
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"KD &Co","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"A01020001","item_name":"HP","bom_no":"BOM-A01020001-001","qty_to_manufacture":400,"work_in_progress_warehouse":"Stores - KD","rejection_quantity":1,"account":"Scrap Account - KD","date":"2020-08-04"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-04 12:40:22,605 ERROR frappe New Exception collected with id: 2020-08-04 12:40:22.464488-127.0.0.1-644
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"KD &Co","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"A01020001","item_name":"HP","bom_no":"BOM-A01020001-001","qty_to_manufacture":400,"work_in_progress_warehouse":"Stores - KD","rejection_quantity":4,"rejection_cause":"Machine miss-alignment","scrap_warehouse":"Scrap Warehouse - KD"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-04 12:41:25,390 ERROR frappe New Exception collected with id: 2020-08-04 12:41:25.244042-127.0.0.1-211
Site: erpnext_dev
Form Dict: {'doc': '{"docstatus":0,"doctype":"Quality Rejection","name":"New Quality Rejection 1","__islocal":1,"__unsaved":1,"owner":"Administrator","company":"KD &Co","select_work_order_number":"MFG-WO-2020-00001","item_to_manufacture":"A01020001","item_name":"HP","bom_no":"BOM-A01020001-001","qty_to_manufacture":400,"work_in_progress_warehouse":"Stores - KD","rejection_quantity":4,"rejection_cause":"Machine miss-alignment","scrap_warehouse":"Scrap Warehouse - KD"}', 'action': 'Save', 'cmd': 'frappe.desk.form.save.savedocs'}
2020-08-04 14:27:54,182 ERROR frappe New Exception collected with id: 2020-08-04 14:27:54.002076-127.0.0.1-dbe
Site: erpnext_dev
Form Dict: {'doctype': 'Quality Rejection', 'name': 'QR-00016', 'cmd': 'frappe.client.get'}
2020-08-04 17:21:15,502 ERROR frappe New Exception collected with id: 2020-08-04 17:21:15.308317-127.0.0.1-3d2
Site: erpnext_dev
Form Dict: {'cmd': 'frappe.realtime.get_user_info'}
